summingbird-core/src/main/scala/com/twitter/summingbird/Options.scala

```diff
@@ -26,6 +26,18 @@ import scala.reflect.{ classTag, ClassTag }
 
 object Options {
   def apply(opts: Map[Class[_], Any] = Map.empty): Options = new Options(opts)
+  /**
+   * Given a list of names, return the first option, if it exists, from the given options
```

Comment:
should add to the comment that we are returning the name we found with the option